src/clj/webchange/common/audio_parser/recognizer.clj

```diff
@@ -0,0 +1,77 @@
+(ns webchange.common.audio-parser.recognizer
+  (:require
+    [clojure.data.json :as json]
+    [clojure.string :as s]
+    [clojure.java.io :as io]
+    [clojure.java.shell :refer [sh]]))
+
+(def parser-directory "src/clj/webchange/common/audio_parser/rhubarb/")
```

Comment:
I think it's better not to store binaries inside the repository.
Let's just add instructions to REAMDE file to install 'rhubab' library and make it available on PATH like ffmpeg